scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -129,10 +129,13 @@
         Optional upload target for metrics. Valid values are AzureLogAnalytics or IoTHub. Default is AzureLogAnalytics. 
 
     .PARAMETER InitializeWithAgentArtifact
-         Boolean specifying if the iotedge installation should initialize edge agent with the official 1.0 image or the desired artifact. If false, the deployment after installation will start the desired agent artifact.
+        Boolean specifying if the iotedge installation should initialize edge agent with the official 1.0 image or the desired artifact. If false, the deployment after installation will start the desired agent artifact.
 
     .PARAMETER TestStartDelay
-         Tests start after delay for applicable modules
+        Tests start after delay for applicable modules
+
+    .PARAMETER RuntimeLogLevel
+        Optional RuntimeLogLevel for EdgeAgent & EdgeHub (Default: debug)
```

Comment:
If the default is debug, will this work on arm32 devices at the current log rotation configuration?

Reply:
Yup, it should. It just won't keep track as many days back in time.

Reply:
Does log rotation work by capping the size? If so I have no concerns

Reply:
that's correct. it uses docker's log capacity config.